rio/srcrobot/subsystems/indexer.py

```diff
@@ -12,11 +12,19 @@ def __init__(self):
         self.indexerMotor.setInverted(True)
         self.indexerMotor.config_kP(0, 2.0)
 
+        self.indexerDiameter = 0.031
+        self.indexerEncoderCPR = 2048.0
+        self.indexerIntakeVelocity = -(Constants.IndexerConstants.kIndexerIntakeSpeedMS/(math.pi*self.indexerDiameter))*self.indexerEncoderCPR
+        self.indexerShootVelocity = -(Constants.IndexerConstants.kIndexerMaxSpeedMS/(math.pi*self.indexerDiameter))*self.indexerEncoderCPR
+
     def indexerIntake(self):
-        return InstantCommand(lambda: self.indexerMotor.set(TalonSRXControlMode.Velocity, -(Constants.IndexerConstants.kIndexerSpeed/(math.pi*0.031))*2048.0))
+        return self.run(lambda: self.indexerMotor.set(TalonSRXControlMode.Velocity, self.indexerIntakeVelocity))
```

Comment:
Glad to see that these are now RunCommand rather than InstantCommand. Especially since some are being used as a subsystem default command. Nice.